docs/tutorials/tutorial-0-Android.rst

```diff
@@ -0,0 +1,100 @@
+Android Tutorial 0 - Hello, world!
+===================================
+
+In this tutorial, you'll take a really simple "Hello, world!" program written in
+Python, convert it into a native Android project.
+
+Setup
+-----
+
+This tutorial assumes you've read and followed the instructions in
+:doc:`/intro/getting-started`. This Tutorial is for Android Developers and Users!
+
+* A ``tutorial`` directory,
+* An activated Python 3 virtual environment
+* Briefcase installed in that virtual environment
+* We need gradle installed https://gradle.org/
+* The Android SDK and installation guid can be found at https://developer.android.com/studio/index.html
+* Ensure that the path to Android sdk is found.
+* THATS IT YOU ARE READY TO GO!
+
+
+Start a new project Using the Beeware Tool Kit
+---------------------------------------------------
+
+Let's get started by using `the handy template <https://github.com/pybee/briefcase-template>`_ ``briefcase-template``:
+
+.. code-block:: bash
+
+    $ pip install cookiecutter briefcase
+    $ cookiecutter https://github.com/pybee/briefcase-template
+
+This will ask a bunch of questions of you. We'll use an `app_name` of "tutorial_0", and a 
+`formal_name` of "Tutorial 0". Set the other values as you like.You'll now have a few files in this folder, including ``tutorial_0``. 
+
+Check out what the provided ``tutorial_0/app.py`` file contains:
+
+.. code-block:: bash
+
+    (BeeWare)user$ cd tutorial_0
```

Comment:
Not everyone's setup will look like this. It's much easier for people to follow along if you just preface command line instructions with `$` only. 

E.g.: just `$ cd tutorial_0`